tutorials/roostats/rs101_limitexample.py

```diff
@@ -0,0 +1,186 @@
+## \file
+## \ingroup tutorial_roostats
+## \notebook
+## Limits: number counting experiment with uncertainty on both the background rate and signal efficiency.
+##
+## The usage of a Confidence Interval Calculator to set a limit on the signal is illustrated
+##
+## \macro_image
+## \macro_output
+## \macro_code
+##
+## \author Kyle Cranmer
```

Comment:
```suggestion
## \date June 2022
## \authors Artem Busorgin, Kyle Cranmer (C++ version)
```
Please add also the date, and you can also add your name here to get some credits :)